hera_pspec/pspecdata.py

```diff
@@ -1978,6 +2026,8 @@ def get_analytic_covariance(self, key1, key2, M=None, exact_norm=False, pol=Fals
         # Collect all the relevant pieces
 
         C11, C22, C21, C12, P11, S11, P22, S22, P21, S21 = [], [], [], [], [], [], [], [], [], []
+        if model == 'foreground_dependent':
```

Comment:
you should describe in the method docstring what `foreground_dependent` and `outer_product` do

Reply:
Sure.